website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -466,7 +466,7 @@ Next, check out the following:
 [generating-crd]: https://book.kubebuilder.io/reference/generating-crd.html
 [markers]: https://book.kubebuilder.io/reference/markers.html
 [crd-markers]: https://book.kubebuilder.io/reference/markers/crd-validation.html
-[memcached_controller]: https://github.com/operator-framework/operator-sdk/blob/v1.3.0/testdata/go/v3/memcached-operator/controllers/memcached_controller.go
+[memcached_controller]: https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v3/memcached-operator/controllers/memcached_controller.go
```

Comment:
Why we will point out the master and not a tag? Should we not bump here the latest tag release?

Reply:
yeah, agreed. master is more prone to changes. I have changed this to point `latest` tag instead